frontend/javascripts/admin/auth/registration_form_generic.tsx

```diff
@@ -274,28 +278,57 @@ function RegistrationFormGeneric(props: Props) {
           </FormItem>
         </Col>
       </Row>
-      {props.hidePrivacyStatement ? null : (
-        <FormItem
-          name="privacy_check"
-          valuePropName="checked"
-          rules={[
-            {
-              validator: (_, value) =>
-                value
-                  ? Promise.resolve()
-                  : Promise.reject(new Error(messages["auth.privacy_check_required"])),
-            },
-          ]}
-        >
-          <Checkbox>
-            I agree to storage and processing of my personal data as described in the{" "}
-            <a target="_blank" href="/privacy" rel="noopener noreferrer">
-              privacy statement
-            </a>
-            .
-          </Checkbox>
-        </FormItem>
-      )}
+      <div className="registration-form-checkboxes">
+        {props.hidePrivacyStatement ? null : (
+          <FormItem
+            name="privacy_check"
+            valuePropName="checked"
+            rules={[
+              {
+                validator: (_, value) =>
+                  value
+                    ? Promise.resolve()
+                    : Promise.reject(new Error(messages["auth.privacy_check_required"])),
+              },
+            ]}
+          >
+            <Checkbox>
+              I agree to storage and processing of my personal data as described in the{" "}
+              <a target="_blank" href="/privacy" rel="noopener noreferrer">
+                privacy statement
+              </a>
+              .
+            </Checkbox>
+          </FormItem>
+        )}
+        {terms != null && !terms.enabled ? null : (
```

Comment:
Could you please invert the logic here? To me it does not read the intutive way I would have expected :see_no_evil:. Also I think the check box would be rendered when `terms == null` according my understanding of the current code

```suggestion
        {terms && terms.enabled ? (
            < check box component .../>
        ) : null }

```

Edit: Ah I see. Still rendering the checkbox even when `terms == null` is intentional :)

Reply:
exactly. in case the ToS are still being fetched for whatever reason, I dont want it to seem that there are none